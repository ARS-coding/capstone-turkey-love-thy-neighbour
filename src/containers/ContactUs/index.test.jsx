import React from "react";
import renderer from "react-test-renderer";
import ContactUS from ".";

describe("ContactUS", () =>
    it("renders ContactUS correctly", () => {
        const tree = renderer.create(<ContactUS />).toJSON();
        expect(tree).toMatchSnapshot();
    }));
