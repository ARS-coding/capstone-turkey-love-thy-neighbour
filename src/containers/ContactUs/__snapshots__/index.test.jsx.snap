// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUS renders ContactUS correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="container"
  >
    <p>
      Contact Us
    </p>
  </div>
</div>
`;
